oaknational/Oak-Web-Application
# feat: zero-rating sanity images

## Description

- adds sanity apiHost to config
- i've also proxied the sanity cdn https://github.com/oaknational/Leaf/pull/48
- and added the correct config

## Issue(s)

Fixes #595

## How to test

1. Go to {cloud link}
2. with network tab
3. You should see image requests being served from `.thenational.academy`:
![image](https://user-images.githubusercontent.com/12934669/188130413-414b392e-9e6e-4714-9f8c-09435e79f985.png)



## Checklist
